app/provider/models.py

```diff
@@ -24,3 +24,11 @@ def __str__(self) -> str:
     acronym_en = models.CharField(_(_context, "Acronym (English)"))
     acronym_it = models.CharField(_(_context, "Acronym (Italian)"), null=True, blank=True)
     acronym_rm = models.CharField(_(_context, "Acronym (Romansh)"), null=True, blank=True)
+
+    _legacy_id = models.IntegerField(
+        _(_context, "Legacy ID"),
```

Comment:
Judging from your description in the PR, these changes might be removed at some point. I'm wondering if it would be better then to not have this in here... or is this going to stay?

Another possibility could be to have a separate model in the BOD app which maps the IDS from legacy to new? Like that, we could only delete the BOD app without worrying about leftovers in the rest of the system.

Reply:
This is only needed as long as we need to synchronize the data and I'm not sure if it makes sense to keep the old ID for some reasons. Although removing legacy leftovers with a migration step is not a big deal (simply dropping the column), moving this to a separate table might be worth a thought, since we need to add this `legacy_id` to other tables (e.g. datasets) as well :+1:

Reply:
@msom @schtibe there will likely be parallel operation of both for a longer time. I'm fine with a legacy_id field on the models and as commented above a field that gives information about the source of the data.

Reply:
After thinking again, I would advise against a separate table (in the `bod` application).
1. Removing the app will not delete this table from the DB. We would need an additional (an probably previous) migration step which removes the application. Removing the additional columns can be done in the same commit as removing the app.
2. Adding a separate table will make things unnecessarily more complicated and prone to errors.